packages/components/Icon/src/Icon.types.ts

```diff
@@ -0,0 +1,35 @@
+import { SvgProps } from 'react-native-svg';
+export const iconName = 'Icon';
+
+export interface IFontIconProps
```

Comment:
Typically props inherit from one of the base primitives (ViewProps, TextProps or ImageProps) then have their own properties added.